src/core/lib/FileSignatures.mjs

```diff
@@ -3157,3 +3329,54 @@ function readHuffmanCode(stream, table) {
 
     return codeWithLength & 0xffff;
 }
+
+
+/**
+ * EVTX extractor.
+ *
+ * @param {Uint8Array} bytes
+ * @param {Number} offset
+ * @returns {Uint8Array}
+ */
+export function extractEVTX(bytes, offset) {
+    const stream = new Stream(bytes.slice(offset));
+    stream.moveTo(0x28);
+
+    // Move to first ELFCHNK.
+    const total = stream.readInt(4, "le") - 0x2c;
+    stream.moveForwardsBy(total);
+    while (stream.hasMore()) {
+
+        // Loop through ELFCHNKs.
+        if (stream.getBytes(7).join("") === [0x45, 0x6c, 0x66, 0x43, 0x68, 0x6e, 0x6b].join(""))
```

Comment:
`getBytes` consumes those bytes, so if we don't find this pattern, do we need to move backwards by 7 bytes?